### PR TITLE
generate a 400 error, instead of 500 when receiving an invalid ARN

### PR DESCRIPTION
**Pull request by blentz**

### CODE REVIEW

File: cloudigrade/util/aws.py [hunk 1]
```diff
@@ -8,6 +8,7 @@
 import boto3
 from botocore.exceptions import ClientError
 from django.conf import settings
+from django.core.exceptions import SuspiciousOperation
```
Comment (ilyavw) [inline cloudigrade/util/aws.py]: Not sure if `SuspiciousOperation` really feels right here, even if we're just re-using it for the easy `400` we may inadvertently accidentally catch `InvalidArn` exceptions if we're catching `SuspiciousOperation` in the future when trying to catch actual security issues. I'd encourage you to extend [DRF `ApiException`](http://www.django-rest-framework.org/api-guide/exceptions/#api-reference) instead, maybe [ParseError](http://www.django-rest-framework.org/api-guide/exceptions/#parseerror).

File: cloudigrade/util/aws.py [hunk 2]
```diff
@@ -70,6 +71,12 @@ def is_running(cls, code):
         return code == cls.running.value
 
 
+class InvalidArn(SuspiciousOperation):
```
Comment (ilyavw) [inline cloudigrade/util/aws.py]: This exception is probably better off living in `exceptions.py`.

Comment (blentz) [reply cloudigrade/util/aws.py]: I only see account/exceptions.py, which doesn't seem right. Are you meaning "create a new exceptions.py file" ?

Comment (ilyavw) [reply cloudigrade/util/aws.py]: Yup!

**Pull request merged**